.github/workflows/pipeline.yml

```diff
@@ -15,9 +36,12 @@ jobs:
           node-version: "18.x"
       - name: Install Dependencies
         run: npm ci
+      - name: Test
```

Comment:
Afra, since testing is more resource-intensive, we can first lint the code and then test it. By this way, if linting breaks we will never jump to testing which will decrease and optimize the total amount of pipeline time.